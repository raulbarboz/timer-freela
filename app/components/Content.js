import React from 'react';
import { useSelector } from 'react-redux';
import Entry from './Entry';
import Company from './Company';


function Content() {
    let data = useSelector(state => state.entry);
    return(
        <div className="container">
            <Entry />
            <Company {...data} />
        </div>
    )
}

export default Content;