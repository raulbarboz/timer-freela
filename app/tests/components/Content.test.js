import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Content from '../../components/Content';

test('should render Content correctly', () => {
    const wrapper = shallow(<Content />);
    expect(toJSON(wrapper)).toMatchSnapshot()
})
