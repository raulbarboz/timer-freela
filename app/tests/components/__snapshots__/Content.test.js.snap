// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Content correctly 1`] = `
<div
  className="container"
>
  <Entry
    calc={[Function]}
  />
  <Output
    withPlan=""
    withoutPlan=""
  />
</div>
`;
